pipelines/crc-qe-virtual-fedora.yaml

```
@@ -0,0 +1,201 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: integration-fedora-test
+spec:
+  description: >-
+    This pipeline used Konflux integration test, 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)